include/internal/libfreenect2/depth_packet_processor.h

```diff
@@ -175,6 +175,25 @@ class OpenCLDepthPacketProcessor : public DepthPacketProcessor
 private:
   OpenCLDepthPacketProcessorImpl *impl_;
 };
+
 #endif // LIBFREENECT2_WITH_OPENCL_SUPPORT
+
+class DumpDepthPacketProcessor : public DepthPacketProcessor
```

Comment:
OK, I see now storage is not a problem for Google. :)

P0 table contains the initial phase shift of the device and can only be obtained at run time. X and Z tables are algorithmic helper and are derived from the intrinsic parameters of the IR camera. The intrinsic parameters are factory built-in and obtained at run time but you can provide your own calibration parameters.

More details about the tables: https://github.com/OpenKinect/libfreenect2/blob/master/src/libfreenect2.cpp#L63

If you don't plan to do your own calibration, you can save the tables in `DumpDepthPacketProcessor::load*Table***()`.